tests/tlsconfig_test.go

```diff
@@ -151,7 +151,7 @@ var _ = Describe("TLS config", func() {
 
 	It("should fail if client certificate is not provided", func() {
 		_, _, err := httpGet("https://"+testHostname+"/apis/"+api.Group+"/"+api.Version+"/", "", PortForwardClient)
-		Expect(err).To(MatchError(ContainSubstring("bad certificate")))
```

Comment:
Is it part of bumping the golang version?

Reply:
Yes, otherwise it would fail CI? I think in the new version, the standard library has changed the TLS error message.